aion/config/dev.exs

```diff
@@ -36,8 +36,8 @@ config :phoenix, :stacktrace_depth, 20
 # Configure your database
```

Comment:
No need to commit changes to this file actually as they are generated by docker and are not part of the PR